part3/identifying-high-leverage-activities.qmd

```diff
@@ -0,0 +1,284 @@
+| telco_customer_support_dev.bronze.plans | Service plan catalog containing plan_id, name, type, pricing, data limits, feature flags (unlimited calls/texts), contract requirements, and descriptions |
+| telco_customer_support_dev.bronze.devices | Device catalog with device_id, name, manufacturer, type, pricing (retail/installment), specifications (storage, 5G compatibility), colors, and availability status |
+| telco_customer_support_dev.bronze.promotions | Promotional offers including promo_id, name, discount details (type/value), date ranges, descriptions, and active status |
+| telco_customer_support_dev.bronze.billing | Billing records with billing_id, customer/subscription references, dates, charge breakdowns (base, additional, tax), payment information, and status |
+| telco_customer_support_dev.bronze.usage | Usage metrics containing usage_id, subscription_id, daily usage data (data_usage_mb, voice_minutes, sms_count), and billing cycle information |
+
+
+Unstructured data sources 
+
+| Table | Unstructured Content |
+----------|-------------------------------|
+| telco_customer_support_dev.bronze.knowledge_base | Full content stored as markdown text including FAQs, policies, guides, and procedures |
+| telco_customer_support_dev.bronze.support_tickets | Free-text issue descriptions from customers |
+| telco_customer_support_dev.bronze.support_tickets | Free-text resolution details from support agents |
+
+
+### Instrumentating observability based on personas
+
+TODO: something here about involving stakeholders early in the planning process and providing good information with user interfaces for internal stakeholders? 
+
+In the chapter Principle 3: The technical practices of learning, we talked at length about how to use tracing to provide visibility into the workings of our agentic application. In this section, we want to expand on this section to cover how traces might be used to provide different stakeholders with a view on how the agent is functioning, in a way that addresses what they care about. 
+
+Building for different personas: 
+
+1. C-suite executives: cares about optics - 
+
+- risk and compliance
+- audit trail completeness 
+- overall claims distribution
+
+business metrics
+
+- processing cost per claim 
+- ROI versus traditional processing methods 
+- settlement accuracy
+
+quality and accuracy indicators
+- agent decision accuracy rate compared to human expert reviews
+- escalation rate to human agents
+- quality score trends over time 
+- false positive / negative rates for claim approvals
+
+2. Business users: cares about interpretability, understand how agents are executing different steps in the workflow so they can troubleshoot / provide inputs if needed. Still want autonomy over the workflows as a human-in-the-loop
```

Comment:
```suggestion
2. Business users: cares about interpretability, understand how agents are executing different steps in the workflow so they can troubleshoot / provide inputs if needed. Still want autonomy over the workflows as a human-in-the-loop
```

Nit - I found the term "user" to be a bit confusing here because the other personas (CEO, developer) are internal to the organization. Assuming all of the personas in this section are internal to the organization, none of them are true "users" of the external-facing telco chatbot.

 Should this be "SME"?

Reply:
Good point. Users are external customers. "Business stakeholders" here would be people from the telco company's customer support department, and hence should be SMEs.